modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,32 +7,39 @@
 = Using Data Mover for CSI snapshots
 
 :FeatureName: Data Mover for CSI snapshots
-include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
+The Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+VolumeSnapshotBackups (VSBs) and VolumeSnapshotRestores (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved, by specifying a concurrent number of VSBs and VSRs simultaneously in Progress.
+
+After all async plugin operations are completed, the backup is marked as completed.
+The Data Mover solution uses the Restic option of VolSync.
+
 
 [NOTE]
 =====
 Data Mover supports backup and restore of CSI volume snapshots only.
-
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
 =====
 
 .Prerequisites
 
-* You have verified that the `StorageClass` and `VolumeSnapshotClass` custom resources (CRs) support CSI.
+* You have verified that the `StorageClass` and `VolumeSnapshotClass` Custom Resources (CRs) support CSI.
 
 * You have verified that only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
 
+* You have verified that `volumeSnapshotClass` CR `deletionPolicy` is set to `Retain`.
```

Comment:
@eemcmullan @rayfordj since you closed https://issues.redhat.com/browse/OADP-1681, should I remove the following line "* You have verified that `volumeSnapshotClass` CR `deletionPolicy` is set to `Retain`."

Reply:
Suggestion: You have verified that the `deletionPolicy` of the `VolumeSnapshotClass` CR ...
The capital V in VolumeSnapshotClass is NOT a suggestion. Unlike labels, and other YAML elements, CR names are written in CamelCaps, not snakeCaps.

Reply:
The VolumeSnapshotClass can in fact now be set to retain or delete, the volumes will be set to retain by velero now.  So it's a good suggestion, but not mandatory.